spec/factories/async_transactions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :async_transaction do
+    sequence(:id, 1) { |n| n }
+    user_uuid 'abcd789-6af0-45e1-a9e2-394347af99'
```

Comment:
Perhaps sequence this `user_uuid`, as we are treating this like a user's id?  i.e.

```ruby
sequence(:transaction_id, 100) { |n| "c2fab2b5-6af0-45e1-a9e2-394347af9#{n}" }
```